models/aws_project.rb

```diff
@@ -85,8 +85,40 @@ def determine_region_mappings
     end
   end
 
+  def validate_credentials
+    puts "Validating AWS project credentials."
+    valid = true
+    begin
+      @explorer.get_cost_and_usage(compute_cost_query(DEFAULT_DATE))
+    rescue => error
```

Comment:
Similar to previous comments, there are lots of errors that could be raised here. What if the system running the program has no internet connection? It would be reported as a problem with AWS, instead of raising a different error to say "we go network problems".

Reply:
Makes sense, thanks.